src/OctoshiftCLI.Tests/GithubClientTests.cs

```diff
@@ -542,6 +542,23 @@ public async Task DeleteAsync_Logs_The_Url()
                 m.LogVerbose(It.Is<string>(actualLogMessage => actualLogMessage == expectedLogMessage)));
         }
 
+        [Fact]
+        public async Task GetHeadersAsync_Is_Successful()
+        {
+            // Arrange
+            using var httpClient = new HttpClient(MockHttpHandlerForGet().Object);
+            var githubClient = new GithubClient(_loggerMock.Object, httpClient, PERSONAL_ACCESS_TOKEN);
+
+            var expectedLogMessage = $"RESPONSE ({HttpStatusCode.OK}): {EXPECTED_RESPONSE_CONTENT}";
+
+            // Act
+            await githubClient.GetHeadersAsync("http://example.com");
+
+            // Assert
+            _loggerMock.Verify(m =>
+                m.LogVerbose(It.Is<string>(actualLogMessage => actualLogMessage == expectedLogMessage)));
+        }
```

Comment:
This test is actually not testing if the request is successful it is just verifying that we do log the response!